Neos.Utility.Unicode/Classes/TextIterator.php

```diff
@@ -195,11 +195,11 @@ public function last()
     }
 
     /**
-     * Returns the next elment following the character of the original string
+     * Returns the next element following the character of the original string
      * given by its offset
      *
      * @param integer $offset The offset of the character
-     * @return string The element following this character
+     * @return integer The element following this character
```

Comment:
`string` was right here? Or either the method name or implementation are wrong. I would expect that this method returns the next character in the subject after the one in offset.